eng/pipelines/docs.yml

```diff
@@ -0,0 +1,84 @@
+trigger:
```

Comment:
Is this the correct trigger? I assume the plan is to have this be an independently run-able pipeline?